web/packages/teleterm/src/ui/AccessRequestCheckout/useAccessRequestCheckout.ts

```diff
@@ -408,7 +430,7 @@ export default function useAccessRequestCheckout() {
       useSearchAsRoles: true,
       nextKey: '',
       resourceType: 'namespace',
-      clusterUri,
+      clusterUri: clusterUri,
```

Comment:
Sorry, I forgot to mention that I specifically didn't use the shorthand here so that it's clearly shown what is deprecated. Otherwise the lang server would highlight the whole field and "Go to definition" would go to the field definition, not the place where the `clusterUri` var is defined.